### ISSUE TITLE
Unable to read the edit field labels for the file properties in Microsoft Excel and Microsoft Word

### ISSUE THREAD

**Issue by DavidApps2** [nvaccess/nvda#8855]
When I view or edit the file properties in Microsoft Excel or Microsoft Word (File > Info), NVDA does not read the labels for the edit fields for the properties ("Title", "Tags", and so on).

### Steps to reproduce:
1. In Microsoft Word, open a document or create a new blank document.
2. Press Alt+F, I to open the file info page.
3. Press Tab. NVDA says "Protect Document grouping Protect Document Alt, F, I, P".
4. Press Tab. NVDA says "Inspect Document grouping Before publishing this file, be aware that it contains Check for Issues Alt, F, I, I".
5. Press Tab. NVDA says "Manage Document grouping Check out document or recover unsaved changes. Manage Document Alt, F, I, R".
6. Press tab. NVDA says "Properties grouping Properties Alt, F, I, Q S".
7. Press Tab. NVDA says "edit blank". I expected it to say "Title edit blank". I can press NVDA+Numpad 8 to obtain the "Title" label.
8. Press Tab. NVDA says "edit blank". I expected it to say "Tags edit blank". I can press NVDA+Numpad 8 to obtain the "Tags" label.
9. Press Tab. NVDA says "edit multi line blank". I expected it to say "Comments edit multi line blank". I can press NVDA+Numpad 8 to obtain the "Comments" label.

### Actual behavior:
NVDA does not announce the labels for the edit fields for the file properties when the user navigates between the fields with the Tab or Shift+Tab keys.
### Expected behavior:
I expected NVDA to read the field labels when the user navigates between the fields.
#### NVDA Installed/portable/running from source:
Installed
#### NVDA version:
2018.3.2
#### Windows version:
10.0.17134 Build 17134 (64-bit)
#### Name and version of other software in use when reproducing the issue:
Microsoft Office 365 MSO 16.0.10827.20181 (32-bit)
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
No
[nvda.log](https://github.com/nvaccess/nvda/files/2493809/nvda.log)

**Comment by DavidApps2** [comment]
NVDA does not read the label for the "Password" edit field in Microsoft Skype for Business. NVDA does not read the label for the "Description" edit field when adding alt text for an image in an email message in Microsoft Outlook. Are these likely to be due to the same issue or should I report these issues separately?
[NVDA-does_not_read_the_label_for_the_password_field_in_Skype_for_Business.log](https://github.com/nvaccess/nvda/files/3505832/NVDA-does_not_read_the_label_for_the_password_field_in_Skype_for_Business.log)

**Comment by Qchristensen** [comment]
@DavidApps2 They are likely different issues.  Best to record each in its own issue just in case - we can always link issues if needed, but it's more confusing if one issue references a handful of things which don't turn out to be related.

**Comment by Adriani90** [comment]
This works as expected in MS Office 365 MSO (16.0.12730.20144) 32-Bit with NVDA Version: alpha-20161,4e580290.
So you need to update your office (i.e. from Microsoft Word > File ribbon > Account > Update options).
Closing as works for me.
